pkg/dsync/drwmutex.go

```diff
@@ -154,13 +154,17 @@ func (dm *DRWMutex) lockBlocking(ctx context.Context, id, source string, isReadL
 		}
 	}
 
+	log("lockBlocking %s/%s: lockType %t with opts: %#v\n", id, source, isReadLock, opts)
+
 	retryCtx, cancel := context.WithTimeout(ctx, opts.Timeout)
 	defer cancel()
 
 	for {
 		// cleanup any older state, re-use the lock slice.
 		cleanLocks(locks)
 
+		log("lockState %#v\n", locks)
```

Comment:
Intentional?

Reply:
yes if MINIO_DSYNC_TRACE is enabled this will be logged @klauspost